boards/arm/ti_am62x_sk_m4/doc/index.rst

```diff
@@ -0,0 +1,148 @@
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 400 MHz.
+
+DDR RAM
+-------
+
+The board has 2GB of DDR RAM available. This board configuration
+allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+MCU domain UART (MCU_UART0). The power domains and clocks for the MCU domain
+are set up by the bootloader.
+
+SD Card
+*******
+
+Download TI's official `WIC`_ and flash the WIC file with an etching software
+onto an SD-card. This will boot Linux on the A53 application cores of the EVM.
+These cores will then load the zephyr binary on the M4 core using remoteproc.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/ti_am62x_sk_m4/ti_am62x_sk_m4_defconfig
+
+Flashing
+********
+
+The board can using remoteproc, and uses the OpenAMP resource table to accomplish this.
+
+The testing requires the binary to be copied to the SD card to allow the A53 cores to load it while booting using remoteproc.
+
+To test the M4F core, we build the `hello_world` sample with the following command.
+
+.. code-block:: console
+
+   # From the root of the Zephyr repository
+   west build -p -b ti_am62x_sk_m4 samples/hello_world
+
+This builds the program and the binary is present in the `build/zephyr` directory as `zephyr.elf`.
+
+We now copy this binary onto the SD card in the `/lib/firmware` directory and name it as `am62-mcu-m4f0_0-fw`.
+
+.. code-block:: console
+
+   # Mount the SD card at sdcard for example
+   sudo mount /dev/sdX sdcard
+   # copy the elf to the /lib/firmware directory
+   sudo cp zephyr.elf sdcard/lib/firmware/am62-mcu-m4f0_0-fw
+
+The SD card can now be used for booting. The binary will now be loaded onto the M4F core on boot.
+
+To allow the board to boot using the SD card, set the boot pins to the SD Card boot mode. Refer to `EVM Setup Page`_.
+
+After changing the boot mode, the board should go through the boot sequence on powering up.
+The binary will run and print Hello world to the MCU_UART0 port.
+
+References
+**********
+
+.. _TI AM62X Product Page:
+   https://www.ti.com/product/AM625
+
+.. _WIC:
+   https://dr-download.ti.com/software-development/software-development-kit-sdk/MD-PvdSyIiioq/08.06.00.42/tisdk-default-image-am62xx-evm.wic.xz
+
+.. _AM62x SK EVM TRM:
+   https://www.ti.com/lit/ug/spruiv7/spruiv7.pdf
+
+.. _EVM Setup Page:
+   https://software-dl.ti.com/mcu-plus-sdk/esd/AM62X/08_06_00_18/exports/docs/api_guide_am62x/EVM_SETUP_PAGE.html
+
+.. _CCS Setup Page:
+   https://software-dl.ti.com/mcu-plus-sdk/esd/AM62X/08_06_00_18/exports/docs/api_guide_am62x/CCS_SETUP_PAGE.html
+
+.. _CCS IDE:
+   http://www.ti.com/tool/ccstudio
+
+.. _CCS Debug Handbook:
+   http://processors.wiki.ti.com/index.php/Debug_Handbook_for_CCS#Cortex_M3.2FM4.2FTiva.2FMSP432
```

Comment:
Other links are relevant, but this one seems to be for MSP432? CCS setup page also is not needed? for more information there is mcu-plus-sdk page already provided, looks like these are here since originally you had only CCS based loading support.

Reply:
Removed all unnecessary links.